pkg/storage/config_test.go

```diff
@@ -34,7 +34,7 @@ func TestShouldMakeNewDB(t *testing.T) {
 	}
 
 	postgresConfigComponent := PostgresConfigComponent{}
-	postgresConfigComponent.New(context.Background(), &postgresConfig)
+	postgresConfigComponent.New(context.Background(), &postgresConfig) // nolint
```

Comment:
Rather than `//nolint`, can we have `_, _ = postgresConfigComponent.New(...)` to explicitly ignore the error?